src/pkg/cli/compose/convert.go

```diff
@@ -32,7 +32,10 @@ func ConvertServices(ctx context.Context, c client.Client, serviceConfigs compos
 	config, err := c.ListConfig(ctx)
 	if err != nil {
 		term.Debugf("failed to load config: %v", err)
-		config = &defangv1.Secrets{}
+		return nil, err
```

Comment:
If we do this, then `defang compose config` requires us to login, before we can see what would happen.